### PR TITLE
README for limitrange example

### PR DESCRIPTION
**Pull request by ikehz**

### CODE REVIEW

File: examples/limitrange/README.md [hunk 1]
```diff
@@ -1,4 +1,96 @@
...
+$ kubectl delete -f examples/limitrange/invalid-pod.json
+pods/invalid-pod
+$ kubectl create -f examples/limitrange/invalid-pod.json
+Error from server: Pod "invalid-pod" is forbidden: Minimum CPU usage per pod is 250m, but requested 10m
+```
+
+A valid pod still works:
+
+```bash
+$ kubectl update -f examples/limitrange/valid-pod.json
```
Comment (roberthbailey) [inline examples/limitrange/README.md]: This command gave me an error:

```
$ kubectl update -f valid-pod.json 
Error from server: Pod "valid-pod" is invalid: spec: invalid value '{Volumes:<nil> Containers:[{Name:kubernetes-serve-hostname Image:gcr.io/google_containers/serve_hostname Command:<nil> Args:<nil> WorkingDir: Ports:<nil> Env:<nil> Resources:{Limits:map[cpu:1 memory:6Mi] Requests:map[]} VolumeMounts:<nil> LivenessProbe:<nil> ReadinessProbe:<nil> Lifecycle:<nil> TerminationMessagePath:/dev/termination-log ImagePullPolicy:IfNotPresent SecurityContext:<nil>}] RestartPolicy:Always TerminationGracePeriodSeconds:<nil> ActiveDeadlineSeconds:<nil> DNSPolicy:ClusterFirst NodeSelector:<nil> ServiceAccount: NodeName: HostNetwork:false ImagePullSecrets:<nil>}': may not update fields other than container.image
```

**Comment by roberthbailey** [comment]
One problem as  I went through the example, otherwise looks good.

File: examples/limitrange/README.md (hunk 1 above)
Comment (jlowdermilk) [inline examples/limitrange/README.md]: `kubectl update -f` doesn't actually work. See #10202. I would suggest we change this command to update the cpu with `--patch`.

**Comment by derekwaynecarr** [comment]
I had a large document here open in a existing PR

https://github.com/GoogleCloudPlatform/kubernetes/pull/9264

**Comment by ihmccreery** [comment]
Closing in favor of #9264.  Let's try to get that in ASAP.

**Pull request closed without merging**